build.sbt

```diff
@@ -40,7 +40,6 @@ lazy val specs2Dependency = Seq(
 val catsVersion = "2.+"
 lazy val catsDependency = Seq(
   "org.typelevel" %% "cats-macros" % catsVersion,
-  "org.typelevel" %% "cats-kernel" % catsVersion,
```

Comment:
what's this change for?

Reply:
I was getting a warning about conflicting versions or something like that, and I realized that cats-kernel is pulled in by cats-core, so I didn't need to explicitly depend on it.